V5 Release note - Productivity part (#879)
akeneo/pim-helpcenter: content/updates/productivity/3-A-more-helpful-process-tracker.md

# More helpful process tracker

## See all ongoing jobs
::: meta-data type="Improvement" ee-only="false"
:::

> To give you more visibility and control on all PIM jobs, we added a new permission: `View all jobs in the Process Tracker` to see all jobs of any status under the process tracker.

Author: ```suggestion
To give you more visibility and control over all the PIM jobs, we added a new permission: `View all jobs in the process tracker`. This permission allows you to see all jobs, regardless of who runs them or what their status is in the process tracker.
```

# More helpful process tracker

> ## See all the ongoing jobs

Reviewer: ```suggestion
## See all ongoing jobs
```

::: meta-data type="Improvement" ee-only="false" link-to-doc="../articles/monitor-jobs.html"
:::

> To give you more visibility and control on all PIM jobs, we added a new permission: `View all jobs in the Process Tracker` to see all jobs of any status under the Process Tracker.

Reviewer: ```suggestion
To give you more visibility and control on all PIM jobs, we added a new permission: `View all jobs in the Process Tracker`. This permission allows you to see all jobs, regardless of who run them or what their status is under Process Tracker.
```

Author: It's not a permission type. It's a permission. :)

> To give you more visibility and control on all PIM jobs, we added a new permission: `View all jobs in the Process Tracker` to see all jobs of any status under the Process Tracker.

Author: To give you more visibility and control over all the PIM jobs, we added a new permission: `View all jobs in the Process Tracker`. This permission allows you to see all jobs, regardless of who runs them or what their status is in the process tracker.

> To give you more visibility and control on all PIM jobs, we added a new permission: `View all jobs in the Process Tracker` to see all jobs of any status under the Process Tracker.

Author: ```suggestion
To give you more visibility and control over all the PIM jobs, we added a new permission: `View all jobs in the Process Tracker`. This permission allows you to see all jobs, regardless of who runs them or what their status is in the process tracker.
```

> You will now be able to see the ongoing or previous PIM jobs and, if needed, which users launched them. :wink:

Reviewer: ```suggestion
```
Not needed imho.

## Monitor ongoing jobs progress and time left
::: meta-data type="New" ee-only="false" link-to-doc="../articles/monitor-jobs.html#what-can-you-find-in-a-job-detail"
:::

Whenever you launch a job, a progress bar and an estimation of the remaining time before the job is completed, are now displayed in the process tracker. You can now easily see how long it will take to process a job, to help you better organize your daily tasks.

![Estimation of the time left and progress bar](../img/progress-bar-for-jobs.png)

## Easily access to the detail of your quick export once launched
::: meta-data type="Improvement" ee-only="false" link-to-doc="../articles/quick-export.html#how"
:::

When you launch a quick product export from the product grid, you can now easily jump to this task's progress detail in the process tracker. Indeed, you'll find a link to the job detail embedded in the flash message that confirms the quick export launch.

## Process tracker display enhancements
::: meta-data type="New" ee-only="false" link-to-doc="../articles/monitor-jobs.html#searching-for-a-job"
:::

> Quickly find a launched job thanks to the new `User` filter we introduce in the process tracker grid. It allows you to easily find either your own jobs or the ones launched by one of your colleagues. Neat!

Reviewer: ```suggestion
Quickly find a launched job thanks to the new `User` filter that we introduce in the process tracker grid. It allows you to easily find either your own jobs or those started by one of your colleagues. Neat!
```

> On a side note, you will also notice that you can sort the process tracker grid jobs by users.

Reviewer: ```suggestion
You are also now able to sort the process tracker grid jobs by users.
```

Author: Sounds really weird without any word to introduce this sentence. It's another improvement. We need something like "on a side note" or "in addition".

Author: ```suggestion
Additionally, you are also able to sort the process tracker grid jobs by users.
```

# More helpful process tracker

## See all the ongoing jobs
::: meta-data type="Improvement" ee-only="false" link-to-doc="../articles/monitor-jobs.html"
:::

To give you more visibility and control on all PIM jobs, we added a new permission: `View all jobs in the Process Tracker` to see all jobs of any status under the Process Tracker.

> You will now be able to see the ongoing or previous PIM jobs and, if needed, which users launched them. :wink:

Reviewer 2: You will now be able to see all the PIM jobs that are started or have been completed and, if needed, which users launched them. :wink:
